model/fn-execution/src/main/proto/beam_fn_api.proto

```diff
@@ -155,6 +155,15 @@ message RegisterRequest {
 message RegisterResponse {
 }
 
+// One of the root applications specifying a bundle.
+// See https://s.apache.org/beam-breaking-fusion
+message Application {
+  // The primitive transform and its input to which to pass the element
+  Target target = 1;
```

Comment:
Target is being dropped, stick specifically with ptransform_id and input

Reply:
Done.